### PR TITLE
Add content: How to Find Error Messages with Sourcegraph (Video)

### PR DESCRIPTION
**Pull request by marekweb**
## Type of change

Add new content.

### What should this PR do?

Add a new piece of content: "How To Find Error Messages with Sourcegraph (Video)"

Ticket: [DEVED-90](https://sourcegraph.atlassian.net/browse/DEVED-90)

### What are the acceptance criteria? 

- The content page loads correctly
- The content card displays in the card lists correctly
- Video is embedded correctly and can be played
- All newly added links work correctly
- Front-matter fields are complete and are accurate
- Transcript is present

### How should this PR be tested?

Open the deploy preview and verify that the content renders correctly and meets the acceptance criteria.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.

### CODE REVIEW

File: posts/how-to-find-error-messages-with-sourcegraph.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: How To Find Error Messages with Sourcegraph (Video)
+author: marek-zaluski
+tags: [video, search, sourcegraph]
+description: Learn about finding error messages in your code with Sourcegraph.
+image: https://i3.ytimg.com/vi/M4zlcZJ2fOA/maxresdefault.jpg
```
Comment (ltagliaferri) [inline posts/how-to-find-error-messages-with-sourcegraph.md:6]: Could we bring this image into the repo?

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
This looks great! I checked the search in Sourcegraph and got similar results, and the transcript looks good and reads well. 

The only thing I would say that would be nice would be if we had all the images hosted on the platform, or if you want to add it to the GCP bucket. Unless there is another reason why we are linking to an off-site image?

File: posts/how-to-find-error-messages-with-sourcegraph.md (hunk 1 above)
Comment (marekweb) [reply posts/how-to-find-error-messages-with-sourcegraph.md:6]: Sure, sounds good. The only advantage of the direct URL is that it updates automatically if we go into YT and edit the video's thumbnail. I don't expect that we'll be updating thumbnails often so it's not so relevant as an advantage.

**Comment by marekweb** [comment]
I added the thumbnail images to the repo for both videos.

**Review by ltagliaferri — APPROVED** [review]
Thanks, Marek! I feel slightly better about us having control of the image, but that is neat to know about the YouTube preview image auto-updating.

**Pull request merged**